notebooks/patient_readmission/patient-readmission-explain-scan.ipynb

```diff
@@ -295,7 +295,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.12"
+   "version": "3.6.9"
```

Comment:
did you mean to check this one in?

Reply:
Yeh - that will be an artifact of the fact that my environment uses local python files for the certifai code.  I'm not sure what to do about it (apart from manually fix the notebook in a text editor, which strikes me as dangerous)

Reply:
Actually does it really matter?